README.md

```diff
@@ -70,6 +70,20 @@ public class DirectoryWatchingUtility {
 }
 ```
 
+## Configuration
+
+By default, DirectoryWatcher will try to prevent duplicate events (e.g. Windows will emit duplicate modify events when file is changed). This is done by creating hash for every file encountered and keeping that hash in memory.
```

Comment:
The text looks good but you are missing articles in a few places. It should be:

"By default, DirectoryWatcher will try to prevent duplicate events (e.g. Windows will emit duplicate modify events when **a** file is changed). This is done by creating **a** hash for every file encountered and keeping that hash in memory."